### DISCUSSION TITLE
saf-training/courses/guidance/06

### DISCUSSION THREAD

**Discussion by giscus** [mitre/saf-training#290] [Ideas]
# saf-training/courses/guidance/06

6.1 Components We have our project created and have allowed access to everyone who needs it. Now let's create ourselves a Component. 6.1.1 Consider Your Component's Requirements...

https://mitre.github.io/saf-training/courses/guidance/06.html

**Comment by karikarshivani** [comment]
Please add asterisks for fields in Vulcan forms

**Reply by p-oneil** [reply]
Moved to PR
